src/app/app-routing.module.ts

```diff
@@ -5,12 +5,26 @@ import { RecipeComponent } from './recipes/recipe/recipe.component';
 import { RecipesComponent } from './recipes/recipes.component';
 import { RecipeEditComponent } from './recipes/recipe-edit/recipe-edit.component';
 import { AppRoutes } from './shared/routes.enum';
+import { AuthGuardService } from './auth/auth.guard.service';
 
 export const routes: Routes = [
+  { path: '', redirectTo: AppRoutes.Recipes, pathMatch: 'full' },
   { path: AppRoutes.Login, component: LoginComponent },
-  { path: AppRoutes.Recipes, component: RecipesComponent },
-  { path: AppRoutes.Recipe + '/:id', component: RecipeComponent },
-  { path: AppRoutes.Edit, component: RecipeEditComponent },
+  {
+    path: AppRoutes.Recipes,
+    component: RecipesComponent,
+    canActivate: [AuthGuardService.authGuardFn],
```

Comment:
This may work but is not correct with convetions.

```
canActivate: [AuthGuard]
```

Naming convention doesnt have Service suffixed, as Service and Guards are different things.

then 

```
export class AuthGuard implements CanActivate {}
```

The class should implement `CanActivate` interface, your guarding logic should e held in `canActivate` method